readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -42,15 +42,15 @@ class BaseSphinx(BaseBuilder):
     # an output file, the parsed source files are cached as "doctree pickles".
     sphinx_doctrees_dir = "_build/doctrees"
 
-    # Output directory relative to where the repository was cloned
-    # (e.g. "_readthedocs/<format>")
+    # Output directory relative to $READTHEDOCS_OUTPUT
+    # (e.g. "html", "htmlzip" or "pdf")
     relative_output_dir = None
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.config_file = self.config.sphinx.configuration
-        self.absolute_output_dir = os.path.abspath(
-            os.path.join(self.project_path, self.relative_output_dir)
+        self.absolute_output_dir = os.path.join(
```

Comment:
Maybe name this 
```suggestion
        self.absolute_container_output_dir = os.path.join(
```

or similar, to differentiate it from `_get_absolute_host_output_dir`?

Reply:
I renamed these variables to make them clearer 👍🏼